bin/dump_wiki

```diff
@@ -38,9 +41,72 @@ if (argv.h) {
 // Strip a trailing slash
 argv.host = argv.host.replace(/\/$/, '');
 
-argv.ns = Number(argv.ns);
+// Default global options
+var options = { maxConcurrency : 50, concurrency: 3 };
+
+if (argv.c) {
+    // Get config, or throw exception on error
+    try {
```

Comment:
this block could be made into a function as it's identical to the one from dump_restbase, except a different property name is used.